misc/config_tools/data/generic_board/generic_code/industry/misc_cfg.h

```diff
@@ -7,85 +7,63 @@
 #ifndef MISC_CFG_H
 #define MISC_CFG_H
 
-#define SOS_ROOTFS		"root=/dev/nvme0n1p3 "
-#define SOS_CONSOLE		"console=ttyS3 "
-#define SOS_COM1_BASE		0x2E8U
-#define SOS_COM1_IRQ		3U
-#define SOS_COM2_BASE		0x3F8U
-#define SOS_COM2_IRQ		4U
-
-#define SOS_BOOTARGS_DIFF	"rw " \
-				"rootwait "	\
-				"console=tty0 "	\
-				"consoleblank=0 "	\
-				"no_timer_check "	\
-				"quiet "	\
-				"loglevel=3 "	\
-				"i915.nuclear_pageflip=1 "	\
-				"swiotlb=131072 "	\
-				"maxcpus=4"
-
-
-#define SOS_VM_CONFIG_CPU_AFFINITY	(AFFINITY_CPU(0U) | AFFINITY_CPU(1U) | AFFINITY_CPU(2U) | AFFINITY_CPU(3U))
-#define VM1_CONFIG_CPU_AFFINITY	(AFFINITY_CPU(0U) | AFFINITY_CPU(1U))
-#define VM2_CONFIG_CPU_AFFINITY	(AFFINITY_CPU(2U) | AFFINITY_CPU(3U))
-#define VM3_CONFIG_CPU_AFFINITY	(AFFINITY_CPU(0U) | AFFINITY_CPU(1U))
-#define VM4_CONFIG_CPU_AFFINITY	(AFFINITY_CPU(0U) | AFFINITY_CPU(1U))
-#define VM5_CONFIG_CPU_AFFINITY	(AFFINITY_CPU(0U) | AFFINITY_CPU(1U))
-#define VM6_CONFIG_CPU_AFFINITY	(AFFINITY_CPU(0U) | AFFINITY_CPU(1U))
-#define VM7_CONFIG_CPU_AFFINITY	(AFFINITY_CPU(0U) | AFFINITY_CPU(1U))
+#define SOS_ROOTFS "root=/dev/nvme0n1p3"
+#define SOS_CONSOLE "console=ttyS3"
+#define SOS_COM1_BASE 0x3F8U
+#define SOS_COM1_IRQ 3U
+#define SOS_COM2_BASE 0x2F8U
+#define SOS_COM2_IRQ 4U
 
+#define SOS_BOOTARGS_DIFF                                                                                              \
+	"rw rootwait console=tty0 consoleblank=0 no_timer_check quiet loglevel=3 i915.nuclear_pageflip=1 "             \
+	"swiotlb=131072 maxcpus=4"
```

Comment:
We seem to be setting this `swiotlb` parameter in many places now, can we explain why we need it, what it does and how to determine the right value for it?

I would put this explanation here: https://projectacrn.github.io/latest/user-guides/kernel-parameters.html

Reply:
@NanlinXie  @szhen11 Do you have any document of this specific argument?

Reply:
Agree, update it later.